lib/game.rb

```diff
@@ -1,12 +1,10 @@
 require "CSV"
 class Game
-
-  attr_reader :away_goals, :home_goals, :game_id, :season 
-
+  attr_reader :visitor_goals, :home_goals, :game_id, :season
```

Comment:
Good call on switching to visitor. Not a big deal but makes sense to keep the naming conventions the same